spec/models/omniauth_data_spec.rb

```diff
@@ -0,0 +1,155 @@
+require 'spec_helper'
+
+describe OmniauthData do
```

Comment:
Karen actually made this data, I just copied it from another test... But now I see she forgot to change a couple of lines... I'll change it.